### DISCUSSION TITLE
Dashboard example with Northwind database.

### DISCUSSION THREAD

**Discussion by wezmag** [serenity-is/Serenity#6623] [Tricks and Samples]
Hello, I created an example Serene project and replaced the dashboard with charts using the real data in the Northwind database.
https://github.com/wezmag/SereneDashboardExample
Please share your thoughts.
Thank you.

Version: 6.4.2

**Comment by AliKILIC** [comment]
Very nice

**Comment by gfo2007** [comment]
@wezmag this is incredible. Please do you have any step by step instructions on how to reproduce what you have done? I am still learning javascript.

Thanks.
